src/rest-server/src/models/v2/job/k8s.js

```diff
@@ -255,6 +255,11 @@ const generateTaskRole = (taskRole, labels, config) => {
                   name: 'host-log',
                   mountPath: '/usr/local/pai/logs',
                 },
+                {
+                  name: 'job-ssh-secret-volume',
+                  readOnly: true,
+                  mountPath: '/etc/ssh-secret',
```

Comment:
maybe mount to `/usr/local/pai/secrets`?

Reply:
I think mount to /etc/ssh-secret is fine because I don't think we should use ssh keys elsewhere.

Reply:
Changed mountPath to /usr/local/pai/ssh-secret. Although I think user should not use this key elsewhere.